### PR TITLE
docs(DevServer): expand on `devServer.openPage` usage

### PR DESCRIPTION
**Pull request by digitaljohn**
Changed documentation of `devServer.openPage` based on **unreleased** changes discussed here:
https://github.com/webpack/webpack-dev-server/pull/2266

> DO NOT MERGE until the aforementioned PR is integrated.

### CODE REVIEW

**Comment by EugeneHlushko** [comment]
We have draft PR feature enabled, you can use it next time for such placeholders

**Comment by montogeek** [comment]
dev server change is released?

**Comment by digitaljohn** [comment]
> dev server change is released?

It appears it has been merged in but not released on NPM as of yet?

**Comment by digitaljohn** [comment]
@montogeek The public version of `webpack-dev-server` now has this feature live.

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -804,7 +805,7 @@ T> The browser application name is platform dependent. Don't hard code it in reu
 
 ## `devServer.openPage`
 
-`string`
+`string` `array = []: string`
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]: Please see: https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types
```suggestion
`string` `[string]`
```

**Comment by digitaljohn** [comment]
> Looks good, just minor change according to our writer's guide

Good spot. Thanks.

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**